.vscode/settings.json

```diff
@@ -27,7 +27,7 @@
     "python.testing.unittestEnabled": false,
     "python.testing.pytestEnabled": true,
     "[github-actions-workflow]": {
-        "editor.defaultFormatter": "redhat.vscode-yaml"
+        "editor.defaultFormatter": "esbenp.prettier-vscode"
```

Comment:
how is this change related?

Reply:
I think it got in by mistake, will remove